src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import './styles/style.css';
+import CalculatorDesign from './components/Calculator';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
```

Comment:
- [ ] The App.js component should be class-based as per the project's requirement. In this milestone, the objective is to practice class-based React components. Kindly make changes here. :pray:

Reply:
Thanks for the review. But in this project requirements in Calculator.js, we need to use class.